docs/stream-aggregation.md

```diff
@@ -334,35 +336,190 @@ For example, the following config removes the `:1m_sum_samples` suffix added [to
+  * The `total` handler returns garbage when something other than [counter](https://docs.victoriametrics.com/keyConcepts.html#counter) is passed to the input.
+
+The algorithm of `total` output can be represented visually by querying the source and aggregated data on the one graph, 
+for `total` it will be queries of the following kind (with aggregation interval `1m`):
+
+* `sum(some_counter)` - source data
+* `some_counter:60s_total` - aggregated data
+
+There is the example of the graph with source and aggregated data 
+(with push interval `10s`, aggregation interval `1m` and `by: ["instance"]` in the stream aggregation config):
+
+<img alt="total aggregation" src="stream-aggregation-check-total.png">
+
+You can see that the `total` output is equal to the sum of the source data every aggregation interval.
+
+### increase
+
 * `increase` returns the increase of input [counters](https://docs.victoriametrics.com/keyConcepts.html#counter).
-  The `increase` handler properly handles the input counter resets.
-  The `increase` handler returns garbage when something other than [counter](https://docs.victoriametrics.com/keyConcepts.html#counter) is passed to the input.
-* `count_series` counts the number of unique [time series](https://docs.victoriametrics.com/keyConcepts.html#time-series).
-* `count_samples` counts the number of input [samples](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `sum_samples` sums input [sample values](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `last` returns the last input [sample value](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `min` returns the minimum input [sample value](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `max` returns the maximum input [sample value](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `avg` returns the average input [sample value](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `stddev` returns [standard deviation](https://en.wikipedia.org/wiki/Standard_deviation) for the input [sample values](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `stdvar` returns [standard variance](https://en.wikipedia.org/wiki/Variance) for the input [sample values](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `histogram_bucket` returns [VictoriaMetrics histogram buckets](https://valyala.medium.com/improving-histogram-usability-for-prometheus-and-grafana-bc7e5df0e350)
+  * The `increase` handler properly handles the input counter resets.
+  * The `increase` handler returns garbage when something other than [counter](https://docs.victoriametrics.com/keyConcepts.html#counter) is passed to the input.
+
+The algorithm of `increase` output can be represented visually by querying the source and aggregated data on the one graph,
+for `increase` it will be queries of the following kind (with aggregation interval `1m`):
```

Comment:
The same issue with unclear wording as above. 
Just keep it simple:
```
The results of `increase` with aggregation interval of `1m` is equal to the `increase(some_counter[1m])` query.
For example, see below time series produced by config with aggregation interval `1m` and `by: ["instance"]` and  the regular query:
<pic>

```

Reply:
done